src/electronmon.js

```diff
@@ -161,7 +161,8 @@ module.exports = ({
 
   function startWatcher() {
     return new Promise((resolve) => {
-      const watcher = watch({ root: cwd, patterns });
+      const parentFolder = cwd.replace("\\Host","")
```

Comment:
What is this? This is replacing part of a path? That seems odd and unintuitive.

Reply:
my apologies, that was something for my own project and shouldn't have been part of this pull request, all other commits should be valid though